### ISSUE TITLE
Fix Node.JS import

### ISSUE THREAD

**Issue by d9k** [bent10/json-loose#7]
```
/home/d9k/.nvm/versions/node/v20.8.0/bin/node --nolazy -r ts-node/register/transpile-only parse-supabase-generated-types-with-babel.ts
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/d9k/cr/pr/examples/ts/supabase-demo/node_modules/.pnpm/json-loose@1.2.2/node_modules/json-loose/dist/index.cjs.js from /home/d9k/cr/pr/examples/ts/supabase-demo/parse-supabase-generated-types-with-babel.ts not supported.
index.cjs.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead either rename index.cjs.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /home/d9k/cr/pr/examples/ts/supabase-demo/node_modules/.pnpm/json-loose@1.2.2/node_modules/json-loose/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at Object.<anonymous> (/home/d9k/cr/pr/examples/ts/supabase-demo/parse-supabase-generated-types-with-babel.ts:33:38)
    at Module._compile (/home/d9k/cr/pr/examples/ts/supabase-demo/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.10.3_typescript@3.9.10/node_modules/ts-node/dist/index.js:857:29)
    at Object..ts (/home/d9k/cr/pr/examples/ts/supabase-demo/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.10.3_typescript@3.9.10/node_modules/ts-node/dist/index.js:859:16) {
  code: 'ERR_REQUIRE_ESM'
}
Process exited with code 1
```

**Comment by d9k** [comment]
Was able to import `json-loose` with https://github.com/privatenumber/tsx

**Comment by bent10** [comment]
I have released [`v1.2.3`](https://github.com/bent10/json-loose/releases/tag/v1.2.3) to address this issue. Could you please confirm that it is working well on your end?
